pages/How-to/guide/create-branch.md

```diff
@@ -0,0 +1,29 @@
+## What is Branches in Git ?
+Git Branch is a new or separate version of main repository. A Git project can have more than one branch, branches create another line of deveolpoment in the project. The primary or default branch in Git is the master branch.
```

Comment:
Spelling error for `development`. You spelled it as `deveolpoment`

Reply:
Thanks for the suggestions, Abbey

Reply:
Thanks for the suggestions, Abbey